### ISSUE TITLE
Feat: Disable `Go to Programs` button on `Organization Profile` page if user who represents organization has not created the organization.

### ISSUE THREAD

**Issue by mtreacy002** [anitab-org/bridge-in-tech-web#87]
**Is your feature request related to a problem? Please describe.**
At the moment the login user who represents an organization but has not created the organization profile can click the button `Go to Programs` on the bottom of the Organization Profile page (under `My Organization` Navbar tab) which should've been prevented.

**Describe the solution you'd like**
Disable the `Go to Programs` button on the `My Organization` Organization Profile page if the login user who has declared that they are an organization representative has not created the organization.

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by saideepesh000** [comment]
@mtreacy002, Can I try this.

**Comment by saideepesh000** [comment]
@mtreacy002, I am working on it, just to be clear is it related to the state of the application(like should I need to deal with the Context part of the application).

**Comment by saideepesh000** [comment]
@mtreacy002, after clicking "I am a representative of an organization", how to join a particular organization if it is available. I mean we can see that the organizations are available on "organizations page", but how to join those organizations

**Comment by mtreacy002** [comment]
@saideepesh000 , at the moment, the flow is for user to declare they represent an organization under the `Additional Info` page, then create an organization. Only after these steps are done, the organization will be shown on the `organizations page`. This means, organization under `organizations page` already has someone representing them so there's no need to link them to user (at least for now). 
In the future, it is nice to have a feature that can make it possible for an organization to be linked/unlinked to another user representative who's not its initial representative. Would you like to open an issue (feature request template)? I will label it as `outreach` to discuss how we're going to approach this. Or if you prefer, I can also open the issue.

**Comment by mtreacy002** [comment]
But just want to clarify, the task of this PR is only to make the `Go to programs` button disable when user has not yet created the organization (so when response message is equal to `Organziation does not exist`). As for when the response message is the user not a representative, I had opened issue #86 to handle that.

**Comment by saideepesh000** [comment]
@mtreacy002, please check the gif, is this the way to deal with this problem.
![PR](https://user-images.githubusercontent.com/43727167/95839030-1ed18080-0d60-11eb-8140-078c665bd7e3.gif)

**Comment by mtreacy002** [comment]
That looks good @saideepesh000 🎉🎉🎉. Can you please push the PR so I can check the code? Also, just as alternative, we can have the `Go to Programs` button still showing but in disable mode so that the user knows where they can go later when they want to add a new/view the list of programs they have created. But whichever one you choose, we will make this consistent across the application 😉.

**Comment by saideepesh000** [comment]
> 
> 
> That looks good @saideepesh000 🎉🎉🎉. Can you please push the PR so I can check the code? Also, just as an alternative, we can have the `Go to Programs` button still showing but in disable mode so that the user knows where they can go later when they want to add a new/view the list of programs they have created. But whichever one you choose, we will make this consistent across the application 😉.

Ok, we'll make button disable for this condition.
